external/fv3fit/fv3fit/reservoir/transformers/transformer.py

```diff
@@ -20,3 +27,85 @@ def encode(self, x: Sequence[ArrayLike]) -> np.ndarray:
     @abc.abstractmethod
     def decode(self, x: ArrayLike) -> Sequence[ArrayLike]:
         pass
+
+
+@io.register("do-nothing-transformer")
+class DoNothingAutoencoder(Transformer, Reloadable):
+    _CONFIG_NAME = "mock_transformer.yaml"
+
+    """Useful class for tests. Encode just concatenates input
+    variables. Decode separates them back into individual arrays.
+    """
+
+    def __init__(
+        self, latent_dim_len, original_feature_sizes: Optional[Sequence[int]] = None
```

Comment:
I think it's still needed for `Transformer` objects since the training code uses the `n_latent_dims` property to determine the size of the reservoir matrices.

Reply:
Sorry, I was thinking that instead of a separate `latent_dim_len` attribute, it would be switched to the sum of feature sizes.  You're right about the need for the property to adhere to the base class.